tests/fixtures/scenarios/metadata/atmos.yaml

```diff
@@ -0,0 +1,25 @@
+base_path: "./"
+
+components:
+  terraform:
+    base_path: "../../components/terraform"
+    apply_auto_approve: false
+    deploy_run_init: true
+    init_run_reconfigure: true
+    auto_generate_backend_file: false
+  
+stacks:
+  base_path: "stacks"
+  included_paths:
+    - "deploy/**/*"
+  excluded_paths:
+    - "**/_defaults.yaml"
+  name_pattern: "{stage}"
+
+logs:
+  file: "/dev/stderr"
+  level: Info
+
+schemas:
+  atmos:
+    manifest: "../../schemas/atmos/atmos-manifest/1.0/atmos-manifest.json"
```

Comment:
Atmos will auto load the schema now. Unless we are specifically testing schema validation, let’s remove this as superfluous.

Reply:
Is that true? I'm not sure it's working without it yet:
```
atmos validate stacks
Schema file 'stacks/schemas/atmos/atmos-manifest/1.0/atmos-manifest.json' not found. Configure via:
1. 'schemas.atmos.manifest' in atmos.yaml
2. ATMOS_SCHEMAS_ATMOS_MANIFEST env var
3. --schemas-atmos-manifest flag

Accepts: absolute path, paths relative to base_path, or URL
```

Reply:
where is the remote schema defined? I cant get that to work without specifying it with one of the given options

Reply:
Ah, so because this PR is not yet merged, the remote schema will fail.

We really need to change the remote schema to a local `go:embed`

Reply:
what should we do here? Update remote schema, remove and admin merge without checks passing, or ignore for now?

Reply:
I'd prefer that we fix the schema handling. @aknysh can you implement the `go:embed` this week?